sweet/harnesses/cockroachdb.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2024 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package harnesses
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"golang.org/x/benchmarks/sweet/common"
+	"golang.org/x/benchmarks/sweet/common/log"
+)
+
+type CockroachDB struct{}
+
+func (h CockroachDB) CheckPrerequisites() error {
+	return nil
+}
+
+func (h CockroachDB) Get(gcfg *common.GetConfig) error {
+	// Build against the latest stable release.
+	return gitShallowClone(
+		gcfg.SrcDir,
+		"https://github.com/cockroachdb/cockroach",
+		"release-24.1",
+	)
+}
+
+func (h CockroachDB) Build(cfg *common.Config, bcfg *common.BuildConfig) error {
+	// Build the cockroach binary.
+	// We do this by using the cockroach `dev` tool. The dev tool is a bazel
+	// wrapper normally used for building cockroach, but can also be to
```

Comment:
Nit: "...but can also be used to generate artifacts for building with `go build`."

Reply:
Done.